docs/Running-the-server.md

```diff
@@ -18,12 +18,27 @@ There are currently two modes to run the Impostor server in. The first way is th
 ### Using docker
 
 [![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor?sort=semver)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
-[![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor/edge)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
```

Comment:
Why has this been removed? Please update to `https://img.shields.io/docker/v/aeonlucid/impostor/nightly`.

Reply:
It errored at 'edge' and the nightly was shown in the semver variant, hence why i removed it. It's back 👍